File: Fitness Tracker/Helpers/HalfSizePresentationController.m
//
//  HalfSizePresentationController.m
//  Fitness Tracker
//
//  Created by Maruf Nebil Ogunc on 4.07.2018.
//  Copyright © 2018 Maruf Nebil Ogunc. All rights reserved.
//

#import "HalfSizePresentationController.h"

@implementation HalfSizePresentationController

-(CGRect)frameOfPresentedViewInContainerView{
    return CGRectMake(0, self.containerView.bounds.size.height/2, self.containerView.bounds.size.width, self.containerView.bounds.size.height/2);
}

@end

File: Fitness Tracker/Helpers/HalfSizePresentationController.swift
//
//  HalfSizePresentationController.swift
//  Fitness Tracker
//
//  Created by AKIN on 16.08.2018.
//  Copyright © 2018 Maruf Nebil Ogunc. All rights reserved.
//

import UIKit

class HalfSizePresentationController: UIPresentationController {
    
    override var frameOfPresentedViewInContainerView: CGRect {
        return CGRect(x: 0, y: containerView!.bounds.height / 2, width: containerView!.bounds.width, height: containerView!.bounds.height / 2)
    }

}
